### PR TITLE
Adding operations and bug fixes to support a custom model

### PR DESCRIPTION
**Pull request by eghasemi89**

### CODE REVIEW

File: python/pyxir/frontend/tvm/relay_tools/relay_l0_expr_and_others.py [hunk 1]
```diff
@@ -38,7 +40,45 @@
 #####################
 # RELAY EXPRESSIONS #
 #####################
+@register_relay_2_xlayer_converter_base('image.resize')
+def image_resize(op_name: str, expr: Expr, in_xlayers: List[XLayer]) -> XLayer:
```
Comment (jornt-xilinx) [inline python/pyxir/frontend/tvm/relay_tools/relay_l0_expr_and_others.py:44]: Can you add a test for this function?

File: python/pyxir/frontend/tvm/relay_tools/relay_l0_expr_and_others.py [hunk 2]
```diff
@@ -38,7 +40,45 @@
...
+    """
+    TVM full to XLayer
```
Comment (jornt-xilinx) [inline python/pyxir/frontend/tvm/relay_tools/relay_l0_expr_and_others.py:46]: This is called image.resize

File: python/pyxir/frontend/tvm/relay_tools/relay_l3_math_and_transform.py [hunk 3]
```diff
@@ -40,6 +41,66 @@
 logger = logging.getLogger("pyxir")
 
 
+
+@register_relay_2_xlayer_converter_base('tile')
+def tile(op_name: str, expr: Expr, in_xlayers: List[XLayer]) -> XLayer:
+    """
+    TVM full to XLayer
```
Comment (jornt-xilinx) [inline python/pyxir/frontend/tvm/relay_tools/relay_l3_math_and_transform.py:48]: This operation is called tile

File: python/pyxir/frontend/tvm/relay_tools/relay_l3_math_and_transform.py [hunk 4]
```diff
@@ -40,6 +41,66 @@
...
+
+    Relay
+    -----
+    Type: tvm.relay.full
```
Comment (jornt-xilinx) [inline python/pyxir/frontend/tvm/relay_tools/relay_l3_math_and_transform.py:52]: tvm.relay.tile

File: python/pyxir/frontend/tvm/relay_tools/relay_l3_math_and_transform.py [hunk 5]
```diff
@@ -40,6 +41,66 @@
...
+    newshape    = [ r*s for r,s in zip(reps,input_shape) ]
+
+    logger.debug("tile: {}".format(op_name))
+    X = px.ops.any_op(op_name, in_xlayers, any_shape=newshape, relay_id=[hash(expr)])
+    logger.debug("-- outshape: {}".format(list(X.shapes)))
+    
+    return X
+
+@register_relay_2_xlayer_converter_base('full')
+def full(op_name: str, expr: Expr, in_xlayers: List[XLayer]) -> XLayer:
```
Comment (jornt-xilinx) [inline python/pyxir/frontend/tvm/relay_tools/relay_l3_math_and_transform.py:72]: Can you add a test for this function?

File: python/pyxir/frontend/tvm/relay_tools/relay_l3_math_and_transform.py [hunk 6]
```diff
@@ -40,6 +41,66 @@
 logger = logging.getLogger("pyxir")
 
 
+
+@register_relay_2_xlayer_converter_base('tile')
+def tile(op_name: str, expr: Expr, in_xlayers: List[XLayer]) -> XLayer:
```
Comment (jornt-xilinx) [inline python/pyxir/frontend/tvm/relay_tools/relay_l3_math_and_transform.py:46]: Can you add a test for this function?

File: tests/unit/frontend/tvm/relay_tools/test_relay_l3_math_and_transform.py [hunk 7]
```diff
@@ -404,3 +404,31 @@ def test_zeros_like(self):
...
+        right = relay.reshape(right, [1,-1])
+
+        net = relay.multiply(left, right)
+
+        mod = tvm.IRModule.from_expr(net)
+        params = {}
+        xgraph = xf_relay.from_relay(mod, params)
+        layers = xgraph.get_layers()
+
+        assert layers[0].type[0] == 'Constant'
```
Comment (jornt-xilinx) [inline tests/unit/frontend/tvm/relay_tools/test_relay_l3_math_and_transform.py:434]: Could you add more checks? For example check whether number of layers are correct, the shapes are correct, etc

File: python/pyxir/frontend/tvm/relay_tools/relay_l0_expr_and_others.py (hunk 1 above)
Comment (eghasemi89) [reply python/pyxir/frontend/tvm/relay_tools/relay_l0_expr_and_others.py:44]: Done

File: python/pyxir/frontend/tvm/relay_tools/relay_l0_expr_and_others.py (hunk 2 above)
Comment (eghasemi89) [reply python/pyxir/frontend/tvm/relay_tools/relay_l0_expr_and_others.py:46]: Fixed

File: python/pyxir/frontend/tvm/relay_tools/relay_l3_math_and_transform.py (hunk 3 above)
Comment (eghasemi89) [reply python/pyxir/frontend/tvm/relay_tools/relay_l3_math_and_transform.py]: Fixed

File: python/pyxir/frontend/tvm/relay_tools/relay_l3_math_and_transform.py (hunk 4 above)
Comment (eghasemi89) [reply python/pyxir/frontend/tvm/relay_tools/relay_l3_math_and_transform.py]: Fixed

File: python/pyxir/frontend/tvm/relay_tools/relay_l3_math_and_transform.py (hunk 5 above)
Comment (eghasemi89) [reply python/pyxir/frontend/tvm/relay_tools/relay_l3_math_and_transform.py:72]: Done

File: python/pyxir/frontend/tvm/relay_tools/relay_l3_math_and_transform.py (hunk 6 above)
Comment (eghasemi89) [reply python/pyxir/frontend/tvm/relay_tools/relay_l3_math_and_transform.py:46]: Done

File: tests/unit/frontend/tvm/relay_tools/test_relay_l3_math_and_transform.py (hunk 7 above)
Comment (eghasemi89) [reply tests/unit/frontend/tvm/relay_tools/test_relay_l3_math_and_transform.py:435]: Done

**Pull request merged**